api/health/handler.go

```diff
@@ -45,14 +47,32 @@ func NewGetAndPostHandler(log logging.Logger, reporter Reporter) (http.Handler,
 	return handler, err
 }
 
+type errorMsg struct {
+	Error string `json:"error"`
+}
+
 // NewGetHandler return a health handler that supports GET requests reporting
 // the result of the provided [reporter].
-func NewGetHandler(reporter func() (map[string]Result, bool)) http.Handler {
+func NewGetHandler(reporter func(tags ...string) (map[string]Result, bool)) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		// Make sure the content type is set before writing the header.
 		w.Header().Set("Content-Type", "application/json")
 
-		checks, healthy := reporter()
+		subnetIDs := r.URL.Query()["subnetID"]
```

Comment:
should this be `"subnetIDs"` since we handle more than one? Idk how query value keys are typically named

Reply:
this collects all values associated with `subnetID` keys. e.g `/ext/health?subnetID=29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL&subnetID=11111111111111111111111111111111LpoYY`

AFAIK there are no standards defining this as explained [here](https://hugomartins.io/essays/2021/02/how-to-pass-multiple-values-to-http-query-parameter/).
We keep this as singular name but let multiple values to be received with repetitive keys.